mobile_app_connector/models/__init__.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+import logging
 ##############################################################################
```

Comment:
Can you remove this comment? It's not needed in this init file.

Reply:
I'll let it in the code since every init file have it. Thanks for the feedbacks !